src/libs/actions/Policy.js

```diff
@@ -24,6 +27,18 @@ Onyx.connect({
     },
 });
 
+let lastAccessedWorkspacePolicyID;
+Onyx.connect({
+    key: ONYXKEYS.LAST_ACCESSED_WORKSPACE_POLICY_ID,
+    callback: (val) => {
+        if (!val) {
+            return;
+        }
+
+        lastAccessedWorkspacePolicyID = val;
+    },
+});
+
```

Comment:
Please remove these changes. Was this discussed in your proposal? Did you add these changes to your proposal?

Reply:
This is used in network reconnect.

Reply:
Okay